### DISCUSSION TITLE
Is it possible to handle WM_STATE property?

### DISCUSSION THREAD

**Discussion by maimere** [i3/i3#4985] [Q&A] [answered]
Greeting, everyone!
I am currently trying to handle Google Chrome tabs while being dragged.

Using `xprop -spy`, I noticed the `WM_STATE` property changed while moving the tab around:

- `xprop` output, not dragging the tab:
```
WM_STATE(WM_STATE):
		window state: Normal
```
- `xprop` output, while dragging the tab:
```
WM_STATE(WM_STATE):
		window state: Withdrawn
```

Is it possible to handle the WM_STATE property in i3?
Hypothetically, I am craving for something like this:
`for_window [class="google-chrome state="Withdrawn"] <...>`

**Comment by Airblader** [comment, marked as answer]
Withdrawn means that the window isn't mapped (= visible). This wouldn't do what you want it to do.

The window while being dragged almost certainly is a new, separate window that Chrome creates with override_redirect. This tells the window manager to ignore the window since the client wants to handle it on its own. In other words, the window you see being dragged is a window i3 doesn't even know or care about.

**Reply by maimere** [reply]
Oh, I get it. Thank you for the explanation.
